src/components/ChapterCard.tsx

```diff
@@ -101,8 +102,8 @@ export default function ChapterCard(props: IProps) {
                                 alignItems: 'center',
                                 padding: 2,
                             }}
-                        >
-                            <div style={{ display: 'flex' }}>
+                        >   
```

Comment:
Remove trailing whitespace
```suggestion
                        >
```